In [8]:
import IPython.display
import pandas as pd

## Candidates

Here it is shown all the coworking spaces found in Foursquare API.

In [7]:
IPython.display.IFrame('../output/candidates_map.html' ,width=1000,  height=500)

## Results

Based on the scoring system, here is the top 3 and bottom 3 along with its score per section and total score.

In [11]:
df = pd.read_csv('../data/score.csv')

In [14]:
df.sort_values('total', ascending = False).head(3)

_id                name  \
0  5fafe7f65143de9ede25a20e           Epicenter   
1  5fafe7f65143de9ede25a21f  Terveystalo Piazza   
2  5fafe7f65143de9ede25a220        Four Reasons   

                                            location  design_score  \
0  {'coordinates': [24.94577562322455, 60.1698553...             0   
1  {'coordinates': [24.930631, 60.169916], 'type'...             0   
2  {'coordinates': [24.929534196853634, 60.168788...             0   

   school_score  tech_score  starbucks_score  airport_score  party_score  \
0           535         225              500          433.0         1935   
1           535         225              500          401.0         1935   
2           535         225              500          392.0         1935   

   vegan_score  basket_score  dog_score   total  
0          100            10          0  3738.0  
1          100            10          0  3706.0  
2          100            10          5  3702.0

In [16]:
df.sort_values('total', ascending = False).tail(3)

_id                name  \
66  5fafe7f65143de9ede25a22d         Eckerö Line   
67  5fafe7f65143de9ede25a23e         Moonsoft Oy   
68  5fafe7f65143de9ede25a247  Ahti Business Park   

                                             location  design_score  \
66  {'coordinates': [24.920848013578677, 60.154510...             0   
67  {'coordinates': [24.820141111789763, 60.179604...             0   
68  {'coordinates': [24.774798229336735, 60.166870...             0   

    school_score  tech_score  starbucks_score  airport_score  party_score  \
66             0           0                0          224.0            0   
67             0           0                0          191.0            0   
68             0           0                0          112.0            0   

    vegan_score  basket_score  dog_score  total  
66            0            10          0  234.0  
67            0            10          0  201.0  
68            0            10          0  122.0

## Winner details

Here is our winner. Check [it](https://weareepicenter.com/helsinki/) out.

Latitude: **60.170**
Longitude: **24.946**

In [17]:
df.sort_values('total', ascending = False).head(1)

_id       name  \
0  5fafe7f65143de9ede25a20e  Epicenter   

                                            location  design_score  \
0  {'coordinates': [24.94577562322455, 60.1698553...             0   

   school_score  tech_score  starbucks_score  airport_score  party_score  \
0           535         225              500          433.0         1935   

   vegan_score  basket_score  dog_score   total  
0          100            10          0  3738.0

### Nearby venues


As shown below:
* Employees with kids would be very happy since close to the venue there is a **very near preschool** to leave children during working hours.
* Also Ceo should not be worried about his/her vegan lunch. Several **vegan-friendly restaurants** are nearby.
* After a long exhausting day, teambuilding is essential and employees have a **great offer of night clubs** to hit.
* **Starbucks** are quite scarce in Finland, actually the very [first Starbucks](https://www.helsinkitimes.fi/business/2241-finland-s-first-starbucks-store-opens-at-helsinki-vantaa-airport.html) came out in 2012 and was located at the Airport! Luckily, our Executives can reach one in less than 5  minutes.
* The major cluster of **tech start-ups** are Helsinki region is located at city center, therefore, developers should also be happy.
* Lastly, maintenance guy could enjoy free time by watching **basketball** at several venues. 

In [20]:
IPython.display.IFrame('../output/nearby_map.html' ,width=1000,  height=500)

### Airport location

Initially, we set an ideal travel time to airport of 20 minutes by car, or 40 minutes by public transport.

Unfortunately, Helsinki International Airport is located at Vantaa, about 18 km from Helsinki Center.
But our venue is located in a very good spot to go to airport, specially by public transport, having the train station just around the corner.

* Travel time by car : **26.7** min
* Travel time by train : **33.3** min

In [22]:
IPython.display.IFrame('../output/2airport_map.html' ,width=1000,  height=500)

### Other Venues

On the first map, you may missed a location of a **design company** to please our designer or a **dog groomer** for 'Pepe'.

Unfortunately, our office does not meet the minimum required distance for those two venues. Nevertheless, below you can find the closest venues along with its distance to our office. Not that bad either right?

In [23]:
IPython.display.IFrame('../output/closest_map.html' ,width=1000,  height=500)

### Where is our most important start-up competitor?

And finally, let's find out where is located our successfull competitor **Supercell**. It is 1.8 km away.

In [5]:
IPython.display.IFrame('../output/competitor_map.html' ,width=1000,  height=500)